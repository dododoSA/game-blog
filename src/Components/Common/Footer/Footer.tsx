import styles from 'styles/Common/Footer/Footer.module.scss';

const Footer = () => (
  <footer>
    <div className={styles.copyright}>©︎2021 R</div>
  </footer>
);

export { Footer };
