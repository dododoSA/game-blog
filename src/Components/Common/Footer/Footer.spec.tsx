import renderer from 'react-test-renderer';
import { Footer } from 'Components/Common/Footer/Footer';

test('Footer', () => {
  const component = renderer.create(<Footer />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
