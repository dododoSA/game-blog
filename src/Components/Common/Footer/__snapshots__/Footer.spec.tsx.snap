// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer 1`] = `
<footer>
  <div
    className="copyright"
  >
    ©︎2021 R
  </div>
</footer>
`;
